R/r-lib.R

```diff
@@ -0,0 +1,49 @@
+#' Helpers from the r-lib organisation
+#'
+#' @details
+#'
+#' * `use_lifecycle()`: adds a dependency on the lifecycle
+#'   package and imports the lifecycle SVG badges to the `man/figures`
+#'   folder. See [lifecycle::badge()] for how to incorporate these
+#'   badges in your documentation.
+#'
+#' @name r-lib
+NULL
+
+#' @rdname r-lib
+#' @export
+use_lifecycle <- function() {
+  check_is_package("use_lifecycle()")
+
+  use_package("lifecycle")
+  use_rd_macros("lifecycle")
+
+  dest_dir <- fs::dir_create(fs::path(proj_get(), "man", "figures"))
+  templ_dir <- fs::path_package("usethis", "templates")
```

Comment:
```suggestion
  templ_dir <- path_package("usethis", "templates")
```

We use fs so much I've imported the whole namespace.